__tests__/app.test.js

```diff
@@ -104,6 +104,49 @@ describe("GET /api", () => {
                     expect(body.msg).toBe("Bad request");
                 })
             })
+            describe("/comments", () => {
+                test("200: Comments should be retrieved with the necessary properties in an array of objects", () => {
+                    return request(app)
+                    .get('/api/articles/1/comments')
+                    .expect(200)
+                    .then(({body}) => {
+                        console.log(body);
```

Comment:
Just remember to remove console logs before pushing code please 🙂

Reply:
Refactored.